src/rgw/rgw_lc.cc

```diff
@@ -351,13 +351,13 @@ int RGWLC::bucket_lc_process(string& shard_id)
   string bucket_id = result[2];
   int ret = store->get_bucket_info(obj_ctx, bucket_tenant, bucket_name, bucket_info, NULL, &bucket_attrs);
   if (ret < 0) {
-    ldout(cct, 0) << "LC:get_bucket_info failed" << bucket_name <<dendl;
+    ldout(cct, 0) << "LC:get_bucket_info for " << bucket_name << " failed" << dendl;
     return ret;
   }
 
   ret = bucket_info.bucket.bucket_id.compare(bucket_id) ;
   if (ret !=0) {
-    ldout(cct, 0) << "LC:old bucket id find, should be delete" << bucket_name <<dendl;
+    ldout(cct, 0) << "LC:old bucket id find, " << bucket_name << " should be deleted" << dendl;
```

Comment:
I think this is better: 
`LC:old bucket id found.  " << bucket_name << " should be deleted.`